raiden/utils/upgrades.py

```diff
@@ -166,6 +167,18 @@ def run(self) -> None:
                 f"older."
             )
 
+        if RAIDEN_DB_VERSION >= 27 and file_version <= 26 and file_version > 1:
+            msg = (
+                f"Your Raiden database is version {file_version} and there is no compatible "
+                f"migration to version {RAIDEN_DB_VERSION} available.\n"
+                "You need to either start a new Raiden node with a different account, or "
+                "close and settle all channels, and start over with a fresh database.\n\n"
```

Comment:
I think this is probably not explaining the problem and solution explicitly enough for the average user / developer. 
Maybe we could add a page to the docs and link to that...